lib/vdsm/common/config.py.in

```diff
@@ -481,6 +481,11 @@ parameters = [
             'are either "filter" or "devices". Filter method will use LVM '
             'filter, while device will use LVM devices file. The default '
             'value is "devices".'),
+
+        ('keep_active', '',
+            'In some case, you might want to keep active some lvm. You can list'
```

Comment:
Maybe better to make the comment more descriptive, similar to other sections. Something similar to this:
```
'Comma-separated list of LVM paths that will never be deactivated. 
For instace: "/dev/vg1/lv1,/dev/vg1/lv2,..."'
```